src/assets/wise5/authoringTool/add-branch/add-branch.component.ts

```diff
@@ -13,30 +13,34 @@ import { MatInputModule } from '@angular/material/input';
 import { CommonModule } from '@angular/common';
 import { MatButtonModule } from '@angular/material/button';
 import { FlexLayoutModule } from '@angular/flex-layout';
-import { Choice } from '../../components/match/choice';
 import { SelectStepComponent } from '../../../../app/authoring-tool/select-step/select-step.component';
 import { SelectComponentComponent } from '../../../../app/authoring-tool/select-component/select-component.component';
+import { BranchPathAuthoringComponent } from '../branch-path-authoring/branch-path-authoring.component';
+import { RouterModule } from '@angular/router';
+import { BranchService } from '../../services/branchService';
 
 @Component({
   selector: 'add-branch',
   templateUrl: './add-branch.component.html',
   styleUrls: ['./add-branch.component.scss'],
   standalone: true,
   imports: [
+    BranchPathAuthoringComponent,
     CommonModule,
     FlexLayoutModule,
     MatButtonModule,
     MatFormFieldModule,
     MatInputModule,
     MatSelectModule,
     ReactiveFormsModule,
+    RouterModule,
```

Comment:
Where do we use this import?

Reply:
The RouterModule is required for the cancel button routerlink in the template.